x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CrawlerPolicies, CrawlerRules, CrawlRule, CrawlerDomainFromServer } from './types';
+
+import { crawlerDomainServerToClient, crawlerDataServerToClient } from './utils';
+
+const DEFAULT_CRAWL_RULE: CrawlRule = {
+  id: '-',
+  policy: CrawlerPolicies.Allow,
+  rule: CrawlerRules.regex,
+  pattern: '.*',
+};
+
+describe('crawlerDomainServerToClient', () => {
+  test('return valuess', () => {
```

Comment:
Why are you using `test` here rather than `it`? I think an `it` description would be helpful to understand what all this is doing... like, snake casing to camel casing, etc.

Reply:
This was a holdover from `ent-search`, I've updated it in 7bb09d6